core/src/main/scala/cats/Show.scala

```diff
@@ -27,7 +32,7 @@ object Show {
 
   final case class Shown(override val toString: String) extends AnyVal
   object Shown {
-    implicit def mat[A](x: A)(implicit z: Show[A]): Shown = Shown(z show x)
+    implicit def mat[A](x: A)(implicit z: ContravariantShow[A]): Shown = Shown(z show x)
```

Comment:
An implicit in the repl is like you've imported it. His is in the Shown companion object.